### ISSUE TITLE
Change the way how the Trade/TradeManager Class reacts to Errors

### ISSUE THREAD

**Issue by awillinger** [Jessecar96/SteamBot#126]
As everyone knows, the current Trade Class throws Exceptions to Errors that may occur (Steam down/Inventory private/etc), which causes the Bot to crash.
This is really annoying when you are running a Donation Bot and it's crashing when you aren't online or simply busy to start it again.

Therefore I have made my own Version of it and now it sends Users a Message, for example if their Inventory is private.
This wouldn't be a Problem until there is an Update released on the Trade Class, causing losing that Modifications and doing everything again.

TL;DR It would be better to Message the User at trivial Errors like private Inventories or Steam Outages (and cancelling the Trade).
